src/components/PageHeader/PageHeader.tsx

```diff
@@ -149,7 +150,7 @@ const PageHeader = () => {
               </div>
             </div>
             <div>
-              <p tw="pl-4 ml-4 mb-0 border-l-2 border-[rgba(221,221,221, 0.5)] leading-none first:pl-0 first:ml-0 first:border-0">{`Version ${measureState?.version}`}</p>
+              <p tw="pl-4 ml-4 mb-0 border-l-2 border-[rgba(225,225,225, 1)] leading-none first:pl-0 first:ml-0 first:border-0">{`Version ${measureState?.version}`}</p>
```

Comment:
I'm seeing some pattern with this paragraph here. this could be its own component. if not part of this task we can create a ticket to address this.

Reply:
Its the styling that is repeated, I think we could use Styled Components.